geomesa-nifi-processors/geomesa-nifi-processors-common/src/test/scala/org/geomesa/nifi/processors/validators/SimpleFeatureTypeValidatorTest.scala

```diff
@@ -13,7 +13,7 @@ import org.junit.Assert._
 import org.junit.Test
 import org.mockito.Mockito._
 
-class TestSimpleFeatureTypeValidator {
+class SimpleFeatureTypeValidatorTest {

```

Comment:
This isn't supposed to run, is it?  There's no JUnit runner annotation above^^^.

Reply:
it does run though, i verified that... maybe the annotation is only needed for specs?

Reply:
Hmm... interesting... maybe the Surefire config (where Test.scala files are configured) takes over?